src/Forside/barnehage/SammenligningMedBransje/EkspanderbartSammenligningspanel.tsx

```diff
@@ -73,6 +76,16 @@ export const EkspanderbartSammenligningspanel: FunctionComponent<Props> = ({
     return (
         <div className={classNames('ekspanderbart-sammenligningspanel', className)}>
             <Ekspanderbartpanel
+                onClick={() => {
+                    sendEvent('barnehage ekspanderbart sammenligning', 'klikk', {
+                        panel:
```

Comment:
Hva med bare `panel: sammenligningsType`? Eller tenker dere at vi får noe ekstra ved å inkludere "fravær" på slutten? 🤔

Reply:
vi tenkte å være litt konkret med å slutte med fravær, for "total" var ganske generelt tenkte vi. Men hvis det er for mye, vi kan vurdere å sende med bare "totalt, korttid, langtid"